content/company/strategy/code-graph/code-intelligence/index.md

```diff
@@ -81,50 +82,37 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q3 goals
+### FY22 Q4 goals
```

Comment:
If this section ends up just being a copy of a subset of the content on the code graph page, you could just link back there and then you don't need to update it twice.